x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml

```diff
@@ -229,7 +229,7 @@ setup:
   - gt: {esql.functions.to_long: $functions_to_long}
   - match: {esql.functions.coalesce: $functions_coalesce}
   - gt: {esql.functions.categorize: $functions_categorize}
-  - length: {esql.functions: 137} # check the "sister" test above for a likely update to the same esql.functions length check
+  - length: {esql.functions: 138} # check the "sister" test above for a likely update to the same esql.functions length check
```

Comment:
Do we want to remove the mute on `test {p0=esql/60_usage/Basic ESQL usage output (telemetry) non-snapshot version}` too?